src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -195,6 +214,7 @@ private static ReadOnlyBag<string> ProcessEnabledFeatures(string[] enabledFeatur
                 }
             }
 
+            SendTelemetryForDeactivatedFeatures(list);
             return new ReadOnlyBag<string>(new HashSet<string>(list, StringComparer.OrdinalIgnoreCase));
```

Comment:
We are going to create a `HashSet` anyways, so it's better to reuse that for `SendTelemetryForDeactivatedFeatures`.
```suggestion
            ReadOnlyBag<string> features = new(new HashSet<string>(list, StringComparer.OrdinalIgnoreCase));
            SendTelemetryForDeactivatedFeatures(features);
            return features;
```
The `ReadOnlyBag` type can be used the same as `HashSet` -- it's essentially a readonly HashSet.